doc/api/process.md

```diff
@@ -2515,7 +2515,8 @@ The [`queueMicrotask()`][] API is an alternative to `process.nextTick()` that
 also defers execution of a function using the same microtask queue used to
 execute the then, catch, and finally handlers of resolved promises. Within
 Node.js, every time the "next tick queue" is drained, the microtask queue
-is drained immediately after.
+is drained immediately after. Note that `process.nextTick()` is scheduled
```

Comment:
`process.nextTick` is not scheduled differently, the module's execution is scheduled differently (specifically, it is a microtask). I think we should be careful of our wording here so people don't get confused.

Reply:
Agree. The wording here is a bit off. I would suggest wording like, "The timing of the execution of tasks scheduled with `process.nextTick()` will vary depending on the module type being used."